docs/application/tizen-studio/common-tools/device-manager.md

```diff
@@ -79,27 +89,27 @@ You can install a Tizen application to a connected device with the TPK or WGT fi
 
     The application is installed and launched on the selected device.
 
-**Figure: Context menu for connected devices**
+    **Figure: Context menu for connected devices**
 
-![Context menu for connected devices](./media/device_manager_connected_devices_context_menu.png)
+    ![Context menu for connected devices](./media/device_manager_connected_devices_context_menu.png)
```

Comment:
This image is changed to the sub item of the list 2.

However, the image itself is about the "Installing Application", not about the "select TPK or WGT file".

So I think we can consider this image as the sub item of the section "Installing Application", 
which means the indentation should be as the original.

Would you please let me know your thought?
If you agree, please revert back to the original.

Reply:
@mijin85cho 
I completely agree that the image is not for "select TPK or WGT file".
The image is for "In the device list, right-click the device to which you want to install an application, and select Install app.".
So, can we move this image after step 1 and fix the indentation?

Please let me know your valuable opinion on the same.

Reply:
@Reni-90 I agree that your suggestion is also another way to display the image. As your suggestion is acceptable, if you hope the change the order in that way, you can.

Reply:
Thank you for your opinion. 
I will move this image after step 1.

Reply:
Moved the image after step 1 and removed the figure caption.
Please verify.